proto.json

```diff
@@ -50,7 +50,7 @@
     "torch.jit.ScriptModule": {
       "code": 12
     },
-    "torch._C.Function": {
+    "torch.jit.ScriptFunction": {
```

Comment:
Integers are cheap, but answering all the issues and Slack questions this will cause is expensive. 😆

Could you give `torch.jit.ScriptFunction‘ it’s own id?

Reply:
I also added the ```torch._C.Function``` and kept the same ```code```. It is ok like that?

Reply:
I don't actually see the change you're describing—maybe it didn't get included in the last commit?